include/ibamr/IBImplicitStaggeredHierarchyIntegrator.h

```diff
@@ -150,11 +156,113 @@ class IBImplicitStaggeredHierarchyIntegrator : public IBHierarchyIntegrator
 
 private:
     /*!
-     * \brief Default constructor.
-     *
-     * \note This constructor is not implemented and should not be used.
-     */
-    IBImplicitStaggeredHierarchyIntegrator();
+	 * \brief A StaggeredStokesSolver that does nothing.
```

Comment:
> On Apr 27, 2015, at 1:06 PM, Amneet Bhalla <notifications@github.com> wrote:
> 
> In include/ibamr/IBImplicitStaggeredHierarchyIntegrator.h <https://github.com/IBAMR/IBAMR/pull/6#discussion_r29166867>:
> 
> > @@ -150,11 +156,113 @@ class IBImplicitStaggeredHierarchyIntegrator : public IBHierarchyIntegrator
> >  
> >  private:
> >      /*!
> > -     * \brief Default constructor.
> > -     *
> > -     * \note This constructor is not implemented and should not be used.
> > -     */
> > -    IBImplicitStaggeredHierarchyIntegrator();
> > +	 * \brief A StaggeredStokesSolver that does nothing.
> Where exactly should I reset the Stokes Op and PC? I don't recall the class and the file...
> 

Use INSStaggeredHierarchyIntegrator::setStokesSolver() to set the solver.

There is not currently an API currently for resetting the Stokes operator itself; if you look in the current implementation, it is specialized for standard Stokes problems, although I don't think that this is really a limitation for our present purposes --- you just need to ensure that the modified IB-Stokes operator is used in the solver, and that the modified problem does not require a modification of the RHS. (For what we are doing, it is enough to modify the solver.)